doc/rules/prefer_intl_name.md

```diff
@@ -0,0 +1,86 @@
+# Prefer Intl name
+
+## Rule id
+prefer-intl-name
+
+## Description
+Prefer used pattern `${ClassName}_${ClassMemberName}` for `name` argument in `Intl.message()`, `Intl.plural()`, `Intl.gender()`, `Intl.select()` methods.
+
+### Example
+Bad:
+```dart
+import 'package:intl/intl.dart';
+
+class SomeButtonI18n {
```

Comment:
@andrewst add examples with static functions (like in this https://github.com/dart-lang/intl#messages)

Reply:
fixed

Reply:
fixed